ui/components/app/collectible-details/index.scss

```diff
@@ -20,7 +20,7 @@ $spacer-break-small: 16px;
       flex-direction: row;
     }
 
-    &__info {
+    &-info {
```

Comment:
I think `&__info` is right, according to BEM. Otherwise this could clash with a `collectible-details-info` component in the future.

Reply:
done here: https://github.com/MetaMask/metamask-extension/pull/13048/commits/e0a1b4c80da158cc529475d6cdf7faca39d269d6